src/roaring_priority_queue.c

```diff
@@ -124,7 +124,7 @@ static roaring_bitmap_t *lazy_or_from_lazy_inputs(roaring_bitmap_t *x1,
             void *c;
 
             if ((container_type_2 == BITSET_CONTAINER_TYPE_CODE) &&
-                (container_type_2 != BITSET_CONTAINER_TYPE_CODE)) {
+                (container_type_1 != BITSET_CONTAINER_TYPE_CODE)) {
```

Comment:
gcc found this impossible branch. This is a total guess of what it's supposed to be checking. Please correct me if I'm wrong!